src/@providers/credentials.ts

```diff
@@ -1,6 +1,8 @@
-import { JSONSchemaType } from 'ajv';
+import { JSONSchemaType } from 'npm:ajv';
```

Comment:
I think ajv is in the import map already, this could cause us to use multiple versions because the import map versions should be pinned